tools/pip_install_requirements.py

```diff
@@ -40,7 +41,13 @@ def pip(req_files):
 
 try:
     with status("Verifying Python package dependencies"):
-        pkg_resources.working_set.resolve([Requirement.parse(r) for r in requirements])
+        parsed_requirements = []
+        for r in requirements:
+            try:
+                parsed_requirements.append(Requirement.parse(r))
+            except InvalidRequirement:
```

Comment:
The only thing you can't check is the version, but you can still check whether the package is installed.

Reply:
@stefanv Are you thinking of some kind of magic string manipulation to get the package name to pass to Requirement.parse?

Reply:
I believe that `#egg=` is followed by a valid requirement specifier.

Reply:
Ok I just split it after that.